### DISCUSSION TITLE
Possible to put 2 buttons in one column?

### DISCUSSION THREAD

**Discussion by laz1r** [TanStack/table#3312] [Q&A]
![Screen Shot 2021-06-07 at 9 13 43 PM](https://user-images.githubusercontent.com/81337065/121125411-5ef7de80-c7db-11eb-8d4a-62cca024bdb6.png)

Is it possible to render 2 buttons in the Add/Delete column of the React Table ? 

I am trying to put both  ADD and DELETE buttons in each column as they are needed when data is retrieved for the row.

**Comment by GuptaSiddhant** [comment]
It definitely is. When defining the “Add/Delete” column, pass a custom component to `Cell` prop of the column object. That custom component can contain anything. 

Check https://react-table-ui.js.org/ for quick getting started with React Table 7.
